packages/eslint-plugin-office-addins/test/rules/call-sync-after-load.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ESLintUtils } from '@typescript-eslint/experimental-utils'
+import rule from '../../src/rules/call-sync-after-load';
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('call-sync-after-load', rule, {
+  valid: [ 
+    {
+      code: `
+        var property = worksheet.getItem("sheet");
+        property.load("values");
+        await context.sync();
+        console.log(property.values);`
+    },
+    {
+      code: `
+        var fakeGet = worksheet.notAGetFunction("props");
+        await context.sync();
+        property.load("props");
```

Comment:
Should also have a test case where fakeGet.load("props") is used.

Reply:
Sure, that's the test I wanted to make, just had made a typo

Reply:
Having both would be good (one test for load without any get and one tests for the fake get call)